saferpayofficial.php

```diff
@@ -1,4 +1,7 @@
 <?php
+
+use Invertus\SaferPay\Service\LegacyTranslator;
```

Comment:
Use full namespaces inside the function instead of importing as it will not work in PS 1.6